### DISCUSSION TITLE
Trouble Building Using Bazel

### DISCUSSION THREAD

**Discussion by retired-sc** [cockroachdb/cockroach#83755] [Q&A] [answered]
When I run ./dev doctor. I get the following error:
`cockroach/bin/dev-versions/dev.38 not found, building...
ERROR: --flag_alias is experimental. Set --experimental_enable_flag_alias to true to make use of it. Detected aliases: --flag_alias=crdb_test=//build/toolchains:crdb_test_flag, --flag_alias=crdb_test_off=//build/toolchains:crdb_test_off_flag, --flag_alias=cross=//build/toolchains:cross_flag, --flag_alias=dev=//build/toolchains:dev_flag, --flag_alias=force_build_cdeps=//build/toolchains:force_build_cdeps_flag, --flag_alias=lintonbuild=//build/toolchains:nogo_flag, --flag_alias=nolintonbuild=//build/toolchains:nonogo_explicit_flag, --flag_alias=with_ui=//pkg/ui:with_ui_flag`

**Comment by retired-sc** [comment]
I am on OpenSuse and I made sure to download all the dependencies.

**Comment by retired-sc** [comment, marked as answer]
The error was JAVA_HOME was pointing to JRE and had to give read and write permissions to the cockroach local repo.

**Reply by jordanlewis** [reply]
Glad you figured it out!
